lib/api/config.js

```diff
@@ -39,6 +41,7 @@ module.exports = Class.create({
 				free_accounts: this.usermgr.config.get('free_accounts'),
 				external_users: this.usermgr.config.get('external_user_api') ? 1 : 0,
 				external_user_api: this.usermgr.config.get('external_user_api') || '',
+				custom_live_log_socket_url: this.server.config.get('custom_live_log_socket_url'),
```

Comment:
yeah, I just want to fix this too then I find u have done the job earlier.  That's great.

BTW, would u mind add this  to `README.md`. something like that 

> ### custom_live_log_socket_url
>
>This is an fancy configuration to provide a url for web socket which live log needs to connect.
>Notice this is useful for single-master system behind an LB.
>url is something like this, e.g.
>
>```js
> "custom_live_log_socket_url": "http://127.0.0.1:8888"
> ```
>
>Feel free to change the ip and the port.

tks.

Reply:
Thanks, committed change